src/signals/incident/containers/IncidentContainer/saga.js

```diff
@@ -111,15 +111,24 @@ export function* getPostData(action) {
     handling_message,
   };
 
+  const invalidFields = ['incident_date', 'incident_time_minutes', 'subcategory', 'datetime', 'incident_time_hours', 'description', 'status'];
```

Comment:
Is this a part of the PR? I can't figure it out...

Reply:
It is, otherwise the fix can't be tested, because of the limitations currently in the API (throwing `400` responses because of props that are part of the payload that should not be sent along).